### ISSUE TITLE
Custom shortcuts not reflected in menus, tooltips, shortcut help, command palette

### ISSUE THREAD

**Issue by stgries** [rstudio/rstudio#5256]
A minor thing: I changed the shortcut for "show previous plot" and "show next plot" but when I mouse-over the corresponding buttons in the Plots pane, the default ones (Ctrl-Alt-F11 etc.) are shown, not the new ones.

**Comment by ronblum** [comment]
@stgries Thank you for raising this! We'll review this issue as we continue development of RStudio. 

I see two options:
1) show the current shortcut; or
2) if that's not possible, don't include the shortcut value because it's incorrect when customized.

**Comment by stgries** [comment]
I personally would prefer 1, but yes, 2 is the best fallback option. Thanks!

**Comment by gtritchie** [comment]
Marking as an accessibility bug, since a screen reader may be reading the wrong keyboard shortcut in this situation.

**Comment by kevinushey** [comment]
@gtritchie, this was marked as part of the 1.4 accessibility project but never had a milestone. Does it still meet the bar for 1.4, or is it worth targeting for a separate milestone?

**Comment by gtritchie** [comment]
Moving to 1.5; this has always been broken AFAIK, so definitely doesn't meet the bar now.

**Comment by mikebessuille** [comment]
Moving back to 1.5 as we really need to fix this.  This had been bulk-moved out.

**Comment by jmcphers** [comment]
The Command Palette, at least, does update correctly now; see https://github.com/rstudio/rstudio/issues/8735.

**Comment by kenahoo** [comment]
This is still an issue in RStudio version 1.4.1725:

<img width="721" alt="Screen Shot 2021-09-28 at 9 49 35 AM" src="https://user-images.githubusercontent.com/111939/135112521-59c2c68d-b10b-4a72-b754-47e1bb4115cc.png">

<img width="480" alt="Screen Shot 2021-09-28 at 9 53 10 AM" src="https://user-images.githubusercontent.com/111939/135112544-847a18c8-0eb9-4936-ab23-4c7771a66b22.png">

**Comment by astayleraz** [comment]
Verified in 2022.01.0-daily+199.pro1 remapping works for a command with and without a default command and added automation coverage for those cases. Closing issue.
